modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/fragment/renderer/SpaceSettingsComponentSectionFragmentRenderer.java

```diff
@@ -43,8 +50,24 @@ protected String getModuleName() {
 	protected Map<String, Object> getProps(
 		FragmentRendererContext fragmentRendererContext,
 		HttpServletRequest httpServletRequest) {
+		
+		ThemeDisplay themeDisplay = (ThemeDisplay)httpServletRequest.getAttribute(WebKeys.THEME_DISPLAY);
+		
+		JSONArray companyAvailableLanguagesJSONArray = JSONFactoryUtil.createJSONArray();
```

Comment:
@julienmouraddotcom, we should use the reference here, instead of JSONFactoryUtil, that's why the SF is complaining

```
@Reference
private JSONFactory _jsonFactory;
```